assets/js/components/setup/search-console.js

```diff
@@ -105,7 +105,7 @@ class SearchConsole extends Component {
 						propertyMatches[ 0 ].siteUrl
 					);
 				} else {
-					errorMessage = __( 'We found no verified accounts, would you like to verify this URL?', 'google-site-kit' );
+					errorMessage = __( 'We did not find a Search Console property for this site, would you like to create one?', 'google-site-kit' );
```

Comment:
Can you change the message to `Your site has not yet been added to Search Console. Would you like to add it now?` please, which is found in #600?

Reply:
Done